tests/ignite/handlers/test_pytorch_profiler.py

```diff
@@ -0,0 +1,51 @@
+import glob
+import os
+
+import pytest
+import torch
+
+import ignite.distributed as idist
+from ignite.engine import Engine
+from ignite.handlers import PyTorchProfiler
+
+
+def update_fn(engine, batch):
+    a = torch.empty((2, 3), dtype=torch.int32)
+    b = torch.empty((3, 3), dtype=torch.int32)
+
+    return a + torch.mm(a, b)
+
+
+def get_engine():
+    dummy_trainer = Engine(update_fn)
+    return dummy_trainer
+
+
+def test_get_results(tmp_path):
```

Comment:
I think  you should test firstly when the profiler is not attached to an engine. Secondly, you should test the presence and the absence of the expected keys.